client/src/index.css

```diff
@@ -1,16 +1,54 @@
-*,
-*::before,
-*::after {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-}
-
-html,
-body {
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
+@font-face {
+  font-family: "Open Sans";
+  src: local("Open Sans"), url(./fonts/OpenSans-Regular.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Rubik";
+  src: local("Rubik"),
+    url(./fonts/Rubik-VariableFont_wght.ttf) format("truetype");
+}
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  font-size: 15px;
+  --black-100: #000000;
+  --black-85: #323232;
+  --black-70: #646464;
+  --black-55: #a0a0a0;
+  --black-40: #c8c8c8;
+  --black-30: #dcdcdc;
+  --black-25: #ebebeb;
+  --white-100: #ffffff;
+  --dot-matrix-width: 2rem;
+}
+
+html,
+body {
+  font-family: "Open Sans", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  letter-spacing: 2%;
+  overflow: hidden;
+  position: relative;
+}
+
+html {
+  background: var(--black-85);
+}
+
+body {
+  border-radius: 25px;
+  background: var(--white-100);
+  border: 5px solid var(--black-100);
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;

```

Comment:
you did not deploy your app yet, so I can not see how your app looks but there is a problem for 100vh on mobile phones. I found a nice tool for that. here's an explanation of the problem and also the solution: https://www.npmjs.com/package/react-div-100vh

Reply:
Thank you very much.

From what I gathered, It's not really my usecase as I am relying on min-height and not using exactly 100vh. But I'll investigate further